src/ajax/xhr.js

```diff
@@ -21,17 +21,6 @@ var xhrId = 0,
 	},
 	xhrSupported = jQuery.ajaxSettings.xhr();
 
-// Support: IE9
```

Comment:
I'm for it. And if you look through the history of this block, you might also find some other code that only exists to support it and can now be removed as well.